app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/IListItemComparator.java

```diff
@@ -0,0 +1,26 @@
+package com.futurice.freesound.feature.common.ui.adapter;
+
+import com.futurice.freesound.feature.common.DisplayableItem;
+
+public interface IListItemComparator {
```

Comment:
The convention in this project is to not use the `I` prefix for interfaces. (This is not something that I'm 100% convinced of ... but those are the rules :-) )